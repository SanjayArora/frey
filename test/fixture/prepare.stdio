ACCPTST:STDIO_SKIP_COMPARE
--> Frey version '0.3.3'
--> Will run: [ 'init', 'config', 'prepare' ]
--> {hostname} - init
--> {hostname} - config
--> {hostname} - prepare
Creating private key '{tmpdir}/frey-prepare.pem'
Creating public key '{tmpdir}/frey-prepare.pub'
Creating private encrypted key '{tmpdir}/frey-prepare.pem.cast5'
Found 'terraform' with version '0.6.12'
Found 'terraformInventory' with version '0.6.0'
Found 'pip' with version {{global_pip_version}}
Found 'pyhcl-0.1.5' with version '0.1.5'
Found 'pyhcl-0.2.0' with version '0.2.0'
Found 'ansible' with version '2.0.0'
Done
Running help on all required tools
  --version             show program's version number and exit
usage: terraform [--version] [--help] <command> [<args>]
    version     Prints the Terraform version
  -version
    	print version information and exit
All tools were able to execute
