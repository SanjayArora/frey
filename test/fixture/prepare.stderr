
--> Exiting with error: Error while checking for '{root}/test/scenario/prepare/.git'
--> Details:

undefined
